docs/cloud/release-notes.rst

```diff
@@ -0,0 +1,33 @@
+Release notes for Qiskit Runtime
```

Comment:
We probably don't need this anymore right? Now that we have the `Release Notes` section for actual qiskit-ibm-runtime releases.

Reply:
Makes sense to me